content/400-reference/200-api-reference/100-prisma-schema-reference.mdx

```diff
@@ -2863,7 +2863,8 @@ In [2.17.0](https://github.com/prisma/prisma/releases/tag/2.17.0) and later, Pri
 #### Remarks
 
 - Ignored fields are excluded from the generated Prisma Client
-- You can manually add `@ignore` to a field that you want to exclude from the Prisma Client (for example, a field that you do not want Prisma users to update)
+- Required fields with no default value marked with `@ignore` removes the `create` and `createMany` methods in the Client API.
+- Can be used to prevent a field from being updated, or make it safe to delete from the database in a subsequent migration with zero downtime.
```

Comment:
```suggestion
- Can be used to prevent a field from being updated, or to make it safe to delete a field from the database in a subsequent migration with zero downtime.
```